test/helpers/random.js

```diff
@@ -0,0 +1,61 @@
+import { CarWriter } from '@ipld/car'
```

Comment:
a copy of https://github.com/w3s-project/w3up/blob/695af8b7313a5edc75235cc578d333577d991e92/packages/w3up-client/test/helpers/random.js